_pages/home/updating-to-version-2/updating-to-version-2.md

```diff
@@ -4,11 +4,15 @@ title: Updating to Version 2.0
 
 ## Prerequisites
 
+First you need to upgrade all your @spartacus libs to the lates 1.5.x release. For more details, see (https://sap.github.io/spartacus-docs/release-information/#about-spartacus-releases)
+
+Make sure you also also have the @spartacus/schematics updaed to the lates 1.5.x version. If you don't have the @spartacus/schemnatics in your package.json, you need to add it. @spartacus/schematics is required in further steps. To add it, edit your `package.json` file. In the `devDependencies` section, add @spartacus/schematics with the same version as your other @spartacus/_ dependencies. For example, if your @spartacus/_ dependencies are on version "^1.5.4", the add the entry `"@spartacus/schematics": "^1.5.4",` in `devDependencies`. After, run `yarn install`.
```

Comment:
Typos: 
updaed -> updated
lates -> latest
spartacus/schemnatics -> spartacus/schematics

Reply:
Instead of `@spartacus/_` maybe we can use a `*`: `@spartacus/*`?

Reply:
Also, I think spartacus is part of `dependencies`, not `devDependencies` section?

Reply:
> Typos:
> updaed -> updated
> lates -> latest
> spartacus/schemnatics -> spartacus/schematics

Fixed, thanks!

> Instead of `@spartacus/_` maybe we can use a `*`: `@spartacus/*`?
Good catch.  Fixed. They were @spartacus/*, Prettier changes them to @spartacus/_ upon save.

Reply:
> Also, I think spartacus is part of `dependencies`, not `devDependencies` section?

It's just for schematics.  I though schematics is a dev tools to be applied on the source and thus are part of `devDependencies`.  What's your recommendation?

Reply:
I agree, it should be part of the `devDependencies`.